browser/src/NeovimInstance.ts

```diff
@@ -128,6 +128,12 @@ export class NeovimInstance extends EventEmitter implements INeovimInstance {
                             const bufferLines = args[0][1]
 
                             this.emit("buffer-update", eventContext, bufferLines)
+                        } else if (pluginMethod === "incremental_buffer_update") {
```

Comment:
Sorry you have a merge conflict here - I moved `NeovimInstance.ts` to the `neovim` folder. Most of the code is the same though, so hopefully this can be brought over to that new file without too much pain. Let me know if you want help here.